packages/product-components/src/components/SelectAssetModal/NetworkTabs.tsx

```diff
@@ -0,0 +1,138 @@
+import { AssetLogo, Row, useElevation } from '@trezor/components';
+import {
+    borders,
+    Elevation,
+    mapElevationToBackground,
+    mapElevationToBorder,
+    spacings,
+    spacingsPx,
+    typography,
+} from '@trezor/theme';
+import { SelectAssetNetworkProps, SelectAssetSearchCategoryType } from './SelectAssetModal';
+import styled, { DefaultTheme } from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+interface CheckableTagProps {
+    $variant: 'primary' | 'tertiary';
+    $elevation: Elevation;
+}
+
+const getCheckableTagStyles = (
+    $elevation: Elevation,
+    theme: DefaultTheme,
+    type: CheckableTagProps['$variant'] | 'hover',
+) => {
+    switch (type) {
+        case 'primary':
+            return `
+                background: ${theme.backgroundPrimarySubtleOnElevation1};
+                color: ${theme.textPrimaryDefault};
+                border: 1px solid ${theme.borderSecondary};
+        `;
+        case 'hover':
+            return `
+                background: ${mapElevationToBackground({ $elevation, theme })};
+                color: ${theme.textDefault};
+                border: 1px solid ${mapElevationToBorder({ $elevation, theme })};
+            `;
+        default:
+            return `
+                background: ${mapElevationToBackground({ $elevation, theme })};
+                color: ${theme.textSubdued};
+                border: 1px solid ${mapElevationToBackground({ $elevation, theme })};
+            `;
+    }
+};
+
+// temporary styles, should be replaced in the future
+const CheckableTag = styled.button<CheckableTagProps>`
+    cursor: pointer;
+    border: 0;
+    ${typography.hint};
+    padding: ${spacingsPx.xxs} ${spacingsPx.sm};
+    border-radius: ${borders.radii.full};
+
+    ${({ $elevation, theme, $variant }) => getCheckableTagStyles($elevation, theme, $variant)}
+
+    &:hover {
+        ${({ $elevation, theme, $variant }) =>
+            getCheckableTagStyles($elevation, theme, $variant === 'primary' ? 'primary' : 'hover')}
+    }
+`;
+
+interface NetworkTabsWrapperProps {
+    $elevation: Elevation;
+}
+
+const NetworkTabsWrapper = styled.div<NetworkTabsWrapperProps>`
+    margin-left: -${spacingsPx.md};
+    width: calc(100% + ${spacings.md * 2}px);
```

Comment:
why do you need this complex calculation here? 🤔

Reply:
It is only to make the border under the wrapper full width.